### DISCUSSION TITLE
Extract Speakers Entities

### DISCUSSION THREAD

**Discussion by nhtkid** [openai/whisper#827] [Q&A] [answered]
Dear Community,

I am using OpenAI Whisper API to extract Podcast into transcriptions. 

The result is a blob of text with no punctuation or paragraph. 

Are there any parameters I could use to get a better result?

Ideally, for a typical Podcast conversation, I like the output to be like dialog.

For example, list each entity in front of each sentence.

Guest 1: ......
Guest 2: ......
Guest 1: ......

Any suggestion is welcome.
Thanks all.

Happy New Year 2023

**Comment by glangford** [comment, marked as answer]
This has already been discussed in 
https://github.com/openai/whisper/discussions/104
https://github.com/openai/whisper/discussions/340
https://github.com/openai/whisper/discussions/571
among other places. Search for "diarization" in the Discussions. See also the recently announced project called Podalize.
https://github.com/mave5/podalize

**Comment by Majdoddin** [comment]
Hi @nhtkid @glangford ,
[www.lexicaps.com](http://www.lexicaps.com/) seamlessly adds diarization to Whispers transcription. No 3rd party packages.
Feel free to contact me if you need more features.
Announcement: https://github.com/openai/whisper/discussions/1537
Repo: https://github.com/Majdoddin/lexicaps
